packages/react-native-client/android/src/main/java/io/fishjam/reactnative/RNFishjamClient.kt

```diff
@@ -517,6 +523,14 @@ class RNFishjamClient(
                   "metadata" to track.metadata
                 )
 
+              is VideoTrack ->
```

Comment:
Does exposing new type of video track will not break anything in existing apps? 🤔

Reply:
It should not as other cases should be matched before it. It is linked with the todo above. After LocalVideoTrack is refactored, this can be removed. But as with other cases, I want to cleanup after I have a working example ready.